modules/core/src/test/java/org/apache/ignite/testframework/GridTestUtils.java

```diff
@@ -1701,6 +1701,9 @@ private static boolean areCompatible(Object[] objs, Class[] types) {
      * {@code assertion} closure will be executed {@code retries} + 1 times and
      * thread will spend approximately {@code retries} * {@code retryInterval} sleeping.
      *
+     * Please consider GridTestUtils#waitForCondition(GridAbsPredicate, long)
+     * to make logs clearer and to avoid possible false positive results.
+     *
```

Comment:
Let's drop this method at all.

Reply:
Done. I've made another method, which in general do the same but it's exception-free to make tests clearer, is it acceptable?